nevergrad/optimization/optimizerlib.py

```diff
@@ -1959,7 +1960,7 @@ class NGOpt2(NGOptBase):
 
     def _select_optimizer_cls(self) -> base.OptCls:
         # override because it is different from NGOptBase (bug?)
-        self.has_discrete_not_softmax = self._has_discrete
+        #self.has_discrete_not_softmax = self._has_discrete
```

Comment:
if you want to remove it please just delete this line. Same with the other below
Just FYI this was a hack to stick with the original version

Reply:
Done.

Reply:
Done.